plugin.py

```diff
@@ -20,6 +20,23 @@ class LspVolarPlugin(NpmClientHandler):
     server_directory = 'server'
     server_binary_path = os.path.join(server_directory, 'node_modules', '@volar', 'server', 'out', 'index.js')
 
+    @classmethod
+    def on_client_configuration_ready(cls, configuration: dict) -> None:
+        is_lsp_typescript_enabled = sublime.load_settings("LSP-typescript.sublime-settings").get("enabled")
+        take_over_mode = configuration.get("settings", {}).get("volar.takeOverMode.enabled", "auto") # type: Union[str, bool]
+
+        def dont_start_in_ts_and_js_files():
+            languages = configuration.get("languages", [])
+            languages_without_typescript = list(filter(lambda langDict: langDict.get('languageId') not in ['typescript', "javascript"], languages))
+            configuration['languages'] = languages_without_typescript
+
+        if take_over_mode == "auto" and is_lsp_typescript_enabled:
+            dont_start_in_ts_and_js_files()
+        if take_over_mode == False:
+            dont_start_in_ts_and_js_files()
+        if take_over_mode == True and is_lsp_typescript_enabled:
+            sublime.status_message('LSP-volar: \"volar.takeOverMode.enabled\" is enabled. Disable "LSP-typescript" or "LSP-volar" to avoid duplicate results.')
```

Comment:
Not that ST has much better alternatives but this is pretty bad way of communicating something to the user. 

It's a long sentence that shows up in the status field for just a couple of seconds so there are very high chances that the user either misses it completely or just don't have enough time to read it.

The alternative could be a modal dialog but those are of course annoying in their own way...